webpack.config.js

```diff
@@ -40,6 +40,10 @@ const webpackConfig = {
     path: path.join(__dirname, "assets/build"),
     filename: "[name].js",
     publicPath: "/assets/build"
+  },
+
+  resolve: {
+    alias: {}
   }
```

Comment:
do we need this?

Reply:
oh yeah, probably not, I forgot to check what could be removed, let me see and push again